extensions/mssql/src/objectManagement/utils.ts

```diff
@@ -11,7 +11,7 @@ import * as localizedConstants from './localizedConstants';
 export async function refreshParentNode(context: azdata.ObjectExplorerContext): Promise<void> {
 	if (context) {
 		try {
-			const node = await azdata.objectexplorer.getNode(context.connectionProfile!.id, context.nodeInfo!.nodePath);
+			const node = await azdata.objectexplorer.getNode(context.connectionProfile!.id, context.nodeInfo?.nodePath);
```

Comment:
Could you add this same fix to `refreshNode` below? It has the same non-null assertion for nodeInfo, which seems like it would cause similar failures if you called it on a top-level node.